message.py

```diff
@@ -20,11 +28,16 @@ def clear_message(self) -> None:
 def build_off_command(id: int) -> bytes:
     return b'0,%d,0,0;' % id
 
+
 def build_on_command(id: int, color: int) -> bytes:
     return b'1,%d,%d,0;' % (id, color)
 
+
 def build_flash_command(id: int, color: int) -> bytes:
     return b'2,%d,%d,0;' % (id, color)
 
+
```

Comment:
What exactly is the difference between fade and flash? 

Is fade merely on, but with opacity?